test/integration/default/serverspec/elasticsearch.rb

```diff
@@ -38,6 +38,11 @@
     its(:stdout) { should match(/.*"status" : "green".*/) }
   end
 
+  # time to obey the previous command
+  describe command('sleep 10') do
```

Comment:
Elasticsearch doesn't always come up in zero seconds with a healthy, so this will probably make the tests more likely to fail.